webapp/src/components/sidebar_right/github_items.tsx

```diff
@@ -165,6 +168,7 @@ function GithubItems(props: GithubItemsProps) {
         if (item.milestone) {
             milestone = (
                 <span
+                    title={'Milestone'}
```

Comment:
Well, the point of screen reader support is to present all needed information for the screen reader. Just knowing _that_ an Issue/PR has a Milestone is not as useful as knowing to _which_ Milestone it's assigned to.

Reply:
> I've noticed that the different elements/icons in the RHS PR list view do not have any aria-labels, so hovering over them has no effect. Some examples can be seen in this screenshot, particularly the PR link and the build status icon
@hanzei based on issue Author's comment, seems this PR should make hovering over icons more descriptive. Like what an icon signifies. Your opinion?

Reply:
Yes, the hover information should be descriptive.

Reply:
Looks like we should have the milestone's name here, based on the discussion in this thread